src/containers/ListItemContainer.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import ListItem from "../components/ListItem";
+
+const ListItemContainer = (props) => {
+  return (
+    <section className="itemSection">
+      <h4 className="sectionTitle">
+        {props.title}(<p className="itemCount">{props.listCnt}</p>)
+      </h4>
+      <div className="itemList invisibleScrollbar">
+        {/* props.listType은 done/todo로 구분된다 */}
+        {props.listState.map((item) => {
+          if (item.type === props.listType) {
+            return (
+              <ListItem
+                key={item.id}
+                item={item}
+                listType={props.listType}
+                modType={props.modType}
+                dispatchListState={props.dispatchListState}
+              />
+            );
+          }
+          return;
+        })}
```

Comment:
너무 잘 작성해주셨는데 &&연산자를 사용해볼 수도 있지 않을까 싶어서 코멘트 남깁니다!
```suggestion
        {props.listState.map((item) => {
          return (
            item.type === props.listType && (
              <ListItem
                key={item.id}
                item={item}
                listType={props.listType}
                modType={props.modType}
                dispatchListState={props.dispatchListState}
              />
            )
          );
        })}
```

Reply:
네 맞습니다, &&연산자를 사용하는 것도 좋은 방법이 될 것 같아요!